mongo/integration/unified/crud_helpers.go

```diff
@@ -157,3 +159,15 @@ func createHint(val bson.RawValue) (interface{}, error) {
 	}
 	return hint, nil
 }
+
+func createCommentString(val bson.RawValue) (comment string, err error) {
```

Comment:
[opt] remove named return values. I think it's fine to have them here, but is there a reason you've employed them?